### ISSUE TITLE
Database migrations with multiple SQL files

### ISSUE THREAD

**Issue by joeljoejoseph** [liquibase/liquibase#1317]
## Environment : Development environment for the POC in production rollout
**Liquibase Version**: 3.7.0

**Liquibase Integration & Version**:  Im currently using maven and the command line liquibase

**Database Vendor & Version**: Oracle & 12.1

**Operating System Type & Version**: Liquibase in windows machine - Windows 7 Professional

## Description
I have tried out executing the multiple files execution method in https://docs.liquibase.com/workflows/liquibase-community/multiple-sql-migration.html and which failed for me and if I remove semicolon (;) issue will be resolved. But ideally multiple sql scripts will be in a single sql file and to remove ; is difficult

## Steps To Reproduce
--Master change log:

<databaseChangeLog  xmlns="http://www.liquibase.org/xml/ns/dbchangelog"
                                  xmlns:ext="http://www.liquibase.org/xml/ns/dbchangelog-ext"
                                  xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
                                xsi:schemaLocation="http://www.liquibase.org/xml/ns/dbchangelog-ext              http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-ext.xsd

http://www.liquibase.org/xml/ns/dbchangelog

http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-3.6.xsd"> 

  <include file="changelog14.xml" />
    </databaseChangeLog>

change log 14:

<databaseChangeLog  xmlns="http://www.liquibase.org/xml/ns/dbchangelog"
                                  xmlns:ext="http://www.liquibase.org/xml/ns/dbchangelog-ext"
                                  xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
                                xsi:schemaLocation="http://www.liquibase.org/xml/ns/dbchangelog-ext              http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-ext.xsd

http://www.liquibase.org/xml/ns/dbchangelog

http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-3.6.xsd"> 

   <includeAll  path="Sql"/>

</databaseChangeLog>

Sql file:
create  table  PERSON  (

```
FNAME  varchar(100)  not  null
```

);

## Actual Behavior
![image](https://user-images.githubusercontent.com/69614205/90090028-94a97380-dd40-11ea-8a36-e1b6acd9a7bd.png)

D:\iFlyRes\Liquibase\Liquibasedemo>liquibase update
Starting Liquibase at Thu, 13 Aug 2020 08:24:01 IST (version 3.7.0 built at 2019-07-16 02:26:39)
Unexpected error running Liquibase: ORA-00922: missing or invalid option
 [Failed SQL: (922) create  table  PERSON  (
    FNAME  varchar(100)  not  null
);]
liquibase.exception.MigrationFailedException: Migration failed for change set Sql/4.sql::raw::includeAll:
     Reason: liquibase.exception.DatabaseException: ORA-00922: missing or invalid option
 [Failed SQL: (922) create  table  PERSON  (
    FNAME  varchar(100)  not  null
);]
        at liquibase.changelog.ChangeSet.execute(ChangeSet.java:646)
        at liquibase.changelog.visitor.UpdateVisitor.visit(UpdateVisitor.java:53)
        at liquibase.changelog.ChangeLogIterator.run(ChangeLogIterator.java:83)
        at liquibase.Liquibase.update(Liquibase.java:202)
        at liquibase.Liquibase.update(Liquibase.java:179)
        at liquibase.integration.commandline.Main.doMigration(Main.java:1206)
        at liquibase.integration.commandline.Main.run(Main.java:192)
        at liquibase.integration.commandline.Main.main(Main.java:130)
Caused by: liquibase.exception.DatabaseException: ORA-00922: missing or invalid option
 [Failed SQL: (922) create  table  PERSON  (
    FNAME  varchar(100)  not  null
);]
        at liquibase.executor.jvm.JdbcExecutor$ExecuteStatementCallback.doInStatement(JdbcExecutor.java:402)
        at liquibase.executor.jvm.JdbcExecutor.execute(JdbcExecutor.java:59)
        at liquibase.executor.jvm.JdbcExecutor.execute(JdbcExecutor.java:131)
        at liquibase.database.AbstractJdbcDatabase.execute(AbstractJdbcDatabase.java:1273)
        at liquibase.database.AbstractJdbcDatabase.executeStatements(AbstractJdbcDatabase.java:1255)
        at liquibase.changelog.ChangeSet.execute(ChangeSet.java:609)
        ... 7 common frames omitted
Caused by: java.sql.SQLSyntaxErrorException: ORA-00922: missing or invalid option

```
    at oracle.jdbc.driver.T4CTTIoer.processError(T4CTTIoer.java:439)
    at oracle.jdbc.driver.T4CTTIoer.processError(T4CTTIoer.java:395)
    at oracle.jdbc.driver.T4C8Oall.processError(T4C8Oall.java:802)
    at oracle.jdbc.driver.T4CTTIfun.receive(T4CTTIfun.java:436)
    at oracle.jdbc.driver.T4CTTIfun.doRPC(T4CTTIfun.java:186)
    at oracle.jdbc.driver.T4C8Oall.doOALL(T4C8Oall.java:521)
    at oracle.jdbc.driver.T4CStatement.doOall8(T4CStatement.java:194)
    at oracle.jdbc.driver.T4CStatement.executeForRows(T4CStatement.java:1000)
    at oracle.jdbc.driver.OracleStatement.doExecuteWithTimeout(OracleStatement.java:1307)
    at oracle.jdbc.driver.OracleStatement.executeInternal(OracleStatement.java:1882)
    at oracle.jdbc.driver.OracleStatement.execute(OracleStatement.java:1847)
    at oracle.jdbc.driver.OracleStatementWrapper.execute(OracleStatementWrapper.java:301)
    at liquibase.executor.jvm.JdbcExecutor$ExecuteStatementCallback.doInStatement(JdbcExecutor.java:398)
    ... 12 common frames omitted
```

For more information, please use the --logLevel flag

## Expected/Desired Behavior
Without removing the semicolon it should work but now its showing error:

## Additional Context
Just tried out the updatesql file to check any difference. First one without semicolon and next one is with semicolon

D:\iFlyRes\Liquibase\Liquibasedemo>liquibase updateSql
Starting Liquibase at Thu, 13 Aug 2020 08:41:24 IST (version 3.7.0 built at 2019-07-16 02:26:39)
-- *********************************************************************
-- Update Database Script
-- *********************************************************************
-- Change Log: master-changelog.xml
-- Ran at: 13/8/20 8:41 AM
-- Against: IRES_LIQUIBASE@jdbc:oracle:thin:@192.168.45.28:1521:ires4528
-- Liquibase version: 3.7.0
-- *********************************************************************

SET DEFINE OFF;

-- Lock Database
UPDATE IRES_LIQUIBASE.DATABASECHANGELOGLOCK SET LOCKED = 1, LOCKEDBY = 'E3444 (10.110.13.86)', LOCKGRANTED = TO_TIMESTAM
P('2020-08-13 08:41:25.678', 'YYYY-MM-DD HH24:MI:SS.FF') WHERE ID = 1 AND LOCKED = 0;

-- Release Database Lock
UPDATE IRES_LIQUIBASE.DATABASECHANGELOGLOCK SET LOCKED = 0, LOCKEDBY = NULL, LOCKGRANTED = NULL WHERE ID = 1;

Liquibase command 'updateSql' was executed successfully.

D:\iFlyRes\Liquibase\Liquibasedemo>liquibase updateSql
Starting Liquibase at Thu, 13 Aug 2020 08:42:56 IST (version 3.7.0 built at 2019-07-16 02:26:39)
-- *********************************************************************
-- Update Database Script
-- *********************************************************************
-- Change Log: master-changelog.xml
-- Ran at: 13/8/20 8:42 AM
-- Against: IRES_LIQUIBASE@jdbc:oracle:thin:@192.168.45.28:1521:ires4528
-- Liquibase version: 3.7.0
-- *********************************************************************

SET DEFINE OFF;

-- Lock Database
UPDATE IRES_LIQUIBASE.DATABASECHANGELOGLOCK SET LOCKED = 1, LOCKEDBY = 'E3444 (10.110.13.86)', LOCKGRANTED = TO_TIMESTAM
P('2020-08-13 08:42:57.316', 'YYYY-MM-DD HH24:MI:SS.FF') WHERE ID = 1 AND LOCKED = 0;

-- Changeset Sql/5.sql::raw::includeAll
create  table  PERSON  (
    FNAME  varchar(100)  not  null
);

INSERT INTO IRES_LIQUIBASE.DATABASECHANGELOG (ID, AUTHOR, FILENAME, DATEEXECUTED, ORDEREXECUTED, MD5SUM, DESCRIPTION, CO
MMENTS, EXECTYPE, CONTEXTS, LABELS, LIQUIBASE, DEPLOYMENT_ID) VALUES ('raw', 'includeAll', 'Sql/5.sql', SYSTIMESTAMP, 12
, '8:20022ec42981301716bae6507f1403fd', 'sql', '', 'EXECUTED', NULL, NULL, '3.7.0', '7288380475');

-- Release Database Lock
UPDATE IRES_LIQUIBASE.DATABASECHANGELOGLOCK SET LOCKED = 0, LOCKEDBY = NULL, LOCKGRANTED = NULL WHERE ID = 1;

Liquibase command 'updateSql' was executed successfully.

D:\iFlyRes\Liquibase\Liquibasedemo>liquibase updateSql
Starting Liquibase at Thu, 13 Aug 2020 08:43:49 IST (version 3.7.0 built at 2019-07-16 02:26:39)
-- *********************************************************************
-- Update Database Script
-- *********************************************************************
-- Change Log: master-changelog.xml
-- Ran at: 13/8/20 8:43 AM
-- Against: IRES_LIQUIBASE@jdbc:oracle:thin:@192.168.45.28:1521:ires4528
-- Liquibase version: 3.7.0
-- *********************************************************************

SET DEFINE OFF;

-- Lock Database
UPDATE IRES_LIQUIBASE.DATABASECHANGELOGLOCK SET LOCKED = 1, LOCKEDBY = 'E3444 (10.110.13.86)', LOCKGRANTED = TO_TIMESTAM
P('2020-08-13 08:43:50.535', 'YYYY-MM-DD HH24:MI:SS.FF') WHERE ID = 1 AND LOCKED = 0;

-- Changeset Sql/5.sql::raw::includeAll
create  table  PERSON  (
    FNAME  varchar(100)  not  null
);

INSERT INTO IRES_LIQUIBASE.DATABASECHANGELOG (ID, AUTHOR, FILENAME, DATEEXECUTED, ORDEREXECUTED, MD5SUM, DESCRIPTION, CO
MMENTS, EXECTYPE, CONTEXTS, LABELS, LIQUIBASE, DEPLOYMENT_ID) VALUES ('raw', 'includeAll', 'Sql/5.sql', SYSTIMESTAMP, 12
, '8:390dbd7a4723f5a9dbf518d3609c8ee2', 'sql', '', 'EXECUTED', NULL, NULL, '3.7.0', '7288433767');

-- Release Database Lock
UPDATE IRES_LIQUIBASE.DATABASECHANGELOGLOCK SET LOCKED = 0, LOCKEDBY = NULL, LOCKGRANTED = NULL WHERE ID = 1;

Liquibase command 'updateSql' was executed successfully.

D:\iFlyRes\Liquibase\Liquibasedemo>liquibase update
Starting Liquibase at Thu, 13 Aug 2020 08:44:07 IST (version 3.7.0 built at 2019-07-16 02:26:39)
Unexpected error running Liquibase: ORA-00922: missing or invalid option
 [Failed SQL: (922) create  table  PERSON  (
    FNAME  varchar(100)  not  null
);]
liquibase.exception.MigrationFailedException: Migration failed for change set Sql/5.sql::raw::includeAll:
     Reason: liquibase.exception.DatabaseException: ORA-00922: missing or invalid option
 [Failed SQL: (922) create  table  PERSON  (
    FNAME  varchar(100)  not  null
);]
        at liquibase.changelog.ChangeSet.execute(ChangeSet.java:646)
        at liquibase.changelog.visitor.UpdateVisitor.visit(UpdateVisitor.java:53)
        at liquibase.changelog.ChangeLogIterator.run(ChangeLogIterator.java:83)
        at liquibase.Liquibase.update(Liquibase.java:202)
        at liquibase.Liquibase.update(Liquibase.java:179)
        at liquibase.integration.commandline.Main.doMigration(Main.java:1206)
        at liquibase.integration.commandline.Main.run(Main.java:192)
        at liquibase.integration.commandline.Main.main(Main.java:130)
Caused by: liquibase.exception.DatabaseException: ORA-00922: missing or invalid option
 [Failed SQL: (922) create  table  PERSON  (
    FNAME  varchar(100)  not  null
);]
        at liquibase.executor.jvm.JdbcExecutor$ExecuteStatementCallback.doInStatement(JdbcExecutor.java:402)
        at liquibase.executor.jvm.JdbcExecutor.execute(JdbcExecutor.java:59)
        at liquibase.executor.jvm.JdbcExecutor.execute(JdbcExecutor.java:131)
        at liquibase.database.AbstractJdbcDatabase.execute(AbstractJdbcDatabase.java:1273)
        at liquibase.database.AbstractJdbcDatabase.executeStatements(AbstractJdbcDatabase.java:1255)
        at liquibase.changelog.ChangeSet.execute(ChangeSet.java:609)
        ... 7 common frames omitted
Caused by: java.sql.SQLSyntaxErrorException: ORA-00922: missing or invalid option

```
    at oracle.jdbc.driver.T4CTTIoer.processError(T4CTTIoer.java:439)
    at oracle.jdbc.driver.T4CTTIoer.processError(T4CTTIoer.java:395)
    at oracle.jdbc.driver.T4C8Oall.processError(T4C8Oall.java:802)
    at oracle.jdbc.driver.T4CTTIfun.receive(T4CTTIfun.java:436)
    at oracle.jdbc.driver.T4CTTIfun.doRPC(T4CTTIfun.java:186)
    at oracle.jdbc.driver.T4C8Oall.doOALL(T4C8Oall.java:521)
    at oracle.jdbc.driver.T4CStatement.doOall8(T4CStatement.java:194)
    at oracle.jdbc.driver.T4CStatement.executeForRows(T4CStatement.java:1000)
    at oracle.jdbc.driver.OracleStatement.doExecuteWithTimeout(OracleStatement.java:1307)
    at oracle.jdbc.driver.OracleStatement.executeInternal(OracleStatement.java:1882)
    at oracle.jdbc.driver.OracleStatement.execute(OracleStatement.java:1847)
    at oracle.jdbc.driver.OracleStatementWrapper.execute(OracleStatementWrapper.java:301)
    at liquibase.executor.jvm.JdbcExecutor$ExecuteStatementCallback.doInStatement(JdbcExecutor.java:398)
    ... 12 common frames omitted
```

For more information, please use the --logLevel flag

**Comment by molivasdat** [comment]
Hi @joeljoejoseph Thanks for writing up this issue.  To re describe your issue.  You have a changelog file that has includeAll in it.
In the directory you have several .sql files or one .sql file?
In that set or single file you have multiple sql statements and if you add a ; it doesn't work and if you do not add a ; it does work?

**Comment by joeljoejoseph** [comment]
Hi,
I have only one sql file and only one sql script. It will work with out ; and fail when give ;

create table PERSON (
FNAME varchar(100) not null
);

**Comment by FBurguer** [comment]
Looks like it got fixed, at least for liquibase `4.17.0`. If you continue to have this issue, feel free to re-open this issue. Thanks!
